content/css/concepts/pseudo-classes/pseudo-classes.md

```diff
@@ -0,0 +1,75 @@
+---
+Title: 'Pseudo-classes'
+Description: A pseudo-class is a keyword added to selectors that specifies a special state of the selected element(s).
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Elements'
+  - 'Classes'
+  - 'Style'
+  - 'Selectors'
+  - 'Properties'
+CatalogContent:
+  - 'learn-css'
+  - 'paths/front-end-engineer-career-path'
+---
+
+A pseudo-class is a keyword added to [selectors](https://www.codecademy.com/resources/docs/css/selectors) that specifies a special state of the selected element(s). For example, you can use it to style an element when a user hovers over it, or when a user clicks it. CSS pseudo-classes can also be used to allow styling based on elements that are in a certain state, such as being disabled or having been visited before.
```

Comment:
```suggestion
A pseudo-class is a keyword added to [selectors](https://www.codecademy.com/resources/docs/css/selectors) that specifies a special state of the selected element(s). For example, it can be used to style an element when a user hovers over it, or when a user clicks it. CSS pseudo-classes can also be used to allow styling based on elements that are in a certain state, such as being disabled or having been visited before.
```

Reply:
@SSwiniarski Good catch!